activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+*   Raise `ActiveRecord::InverseOfAssociationNotBidirectionalError` if invalid inverse_of is specified
+
+    The association specified in `inverse_of` should be the name of the association
+    on the other model that points back to the current model.
+    If the user inadvertently specifies an invalid association name in `inverse_of`
+    that is not bi-directional, the `inverse_of` is implicitly ignored.
+    However, if it remains implicitly ignored, the user can't know that the inverse_of
+    is incorrect. Therefore, I have made a change to raise an exception instead of.
```

Comment:
```suggestion
    This error occurs when the association specified in the `inverse_of` option does not exist on the associated class.
    Previously this would be implicitly ignored, so the developer wouldn't know they tried to make an invalid association.
```